types/react-native/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-native 0.57
+// Type definitions for react-native 0.59
```

Comment:
Was this API only added in 0.59? If not, then there’s no reason to bump this now and exclude this update from people on older RN versions.

Reply:
It was not immediately clear to me, when it would be "appropriate" to update the version, nor that by updating I would be excluding anyone from this update.

```
If changing an existing definition:

- Provide a URL to documentation or source code which provides context for the suggested changes: <>
- Increase the version number in the header if appropriate.
```

The API was added long ago, so I'll take this out.

Reply:
People can install whatever version they want, but that is kind of how what the versions are trying to convey. Thanks!